content/blog/2024/veda-devseed-collab/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Collaborating with DevSeed to deliver cyberinfrastructure for NASA VEDA
```

Comment:
Could we say Development Seed instead of DevSeed in the title? Thank you!!

Reply:
Of course!

Reply:
```suggestion
title: Collaborating with Development Seed to deliver cyberinfrastructure for NASA VEDA
```